noetic/distribution.yaml

```diff
@@ -5460,6 +5460,28 @@ repositories:
       url: https://github.com/pr2/pr2_robot.git
       version: kinetic-devel
     status: unmaintained
+  pr2_self_test:
+    doc:
+      type: git
+      url: https://github.com/PR2/pr2_self_test.git
+      version: kinetic-devel
+    release:
+      packages:
+      - joint_qualification_controllers
+      - pr2_bringup_tests
+      - pr2_counterbalance_check
+      - pr2_motor_diagnostic_tool
+      - pr2_self_test
+      - pr2_self_test_msgs
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/pr2-gbp/pr2_self_test-release.git
+      version: 1.0.15-1
+    source:
+      type: git
+      url: https://github.com/PR2/pr2_self_test.git
+      version: kinetic-devel
+    status: unmaintained
```

Comment:
I'm not sure that releasing an unmaintained packaged would be a good idea. I guess as long as the builds stay green it could be okay, but we should remove it if it starts failing on the buildfarm.